projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -612,28 +609,50 @@ export function selectAttributeAndWait(
   configuration.selectAttribute(attributeName, uiType, valueName, false);
   cy.wait(UPDATE_CONFIG_ALIAS);
   if (isPricingEnabled) {
-    cy.wait(CONFIG_PRICING_ALIAS);
+    waitForPricing();
   }
 }
 
+function waitForPricing() {
```

Comment:
why do we not create one function to handle all the different wait conditions?
something like:
wait(){
  cy.wait(GET_CONFIG_ALIAS);
  if (isPricingEnabled) {
     waitForPricing();
   }
}
this would make it shorter (line 635ff, line 652ff)and easier to maintain for future adaptions  since we call only 

wait() 

here instead of 

cy.wait(GET_CONFIG_ALIAS);
   if (isPricingEnabled) {
     waitForPricing();
   }

Reply:
good pont